setup.py

```diff
@@ -103,6 +103,9 @@
         'Topic :: Internet :: WWW/HTTP :: WSGI :: Server',
     ],
     entry_points={
+        'console_scripts': [
+            'cheroot = cheroot.__main__:main'
```

Comment:
@jaraco what do you think about this script naming? Do we want it to have trailing `d` to be consistent with CherryPy? Does it need a daemon mode?